src/analysis/retail/hunter/beastmastery/normalizers/BarbedShotNormalizer.ts

```diff
@@ -0,0 +1,47 @@
+import SPELLS from 'common/SPELLS';
+import TALENTS from 'common/TALENTS/hunter';
+import { AnyEvent, EventType } from 'parser/core/Events';
+import EventsNormalizer from 'parser/core/EventsNormalizer';
+import { MS_BUFFER_100 } from '../../shared';
+
+export default class BarbedShotNormalizer extends EventsNormalizer {
+  normalize(events: AnyEvent[]) {
+    const fixedEvents: AnyEvent[] = [];
+    const possibleBarbedShotBuffs = [
+      SPELLS.BARBED_SHOT_BUFF.id,
+      SPELLS.BARBED_SHOT_BUFF_2.id,
+      SPELLS.BARBED_SHOT_BUFF_3.id,
+      SPELLS.BARBED_SHOT_BUFF_4.id,
+      SPELLS.BARBED_SHOT_BUFF_5.id,
+      SPELLS.BARBED_SHOT_BUFF_6.id,
+      SPELLS.BARBED_SHOT_BUFF_7.id,
+      SPELLS.BARBED_SHOT_BUFF_8.id,
```

Comment:
NotLikeDuck

Reply:
I wanted to use the EventOrderNormalizer,  but this has to move the event multiple events back so here we are 😢
1 cast = 7 events on the timestamp with the actual cast in the middle NotLikeDuck 
![image](https://user-images.githubusercontent.com/29204244/215609842-65ad0a17-35e3-4615-bc84-f5a89de1c722.png)